CHANGELOG.md

```diff
@@ -1,14 +1,12 @@
 # [1.4.0](https://github.com/BuilderIO/gpt-crawler/compare/v1.3.0...v1.4.0) (2024-01-15)
 
-
 ### Bug Fixes
 
-* linting ([0f4e58b](https://github.com/BuilderIO/gpt-crawler/commit/0f4e58b400eab312e7b595d7a2472bae93055415))
-
+- linting ([0f4e58b](https://github.com/BuilderIO/gpt-crawler/commit/0f4e58b400eab312e7b595d7a2472bae93055415))
```

Comment:
?

Reply:
It occurs to me that it might be due to some local configuration on your machine, @kaibadash. The linter is not  configured to look at markdown files.